n_t/msmc.py

```diff
@@ -60,12 +60,12 @@ def run_msmc_estimate(input_files, output_file, msmc_exec_loc, iterations=1, nco
     """
     
     cmd = (f"{msmc_exec_loc} --fixedRecombination -o \
-        {output_file} -i {iterations} -t {ncores} {input_files}")
```

Comment:
why no more `ncores`?

Reply:
Without this parameter, it uses all available cores, which is more straightforward than `-j` which specifies the number of `rules` which can be run in parallel. But I'm happy to include it.

Reply:
ah definitely more straightforward that `-j`. keep it this way.